Nodejs/Tests/Core.UI/SnippetsTests.cs

```diff
@@ -215,14 +214,12 @@ class Declaration {
             foreach (var decl in snippet.Declarations) {
                 Console.WriteLine("Declaration: {0}", decl.Replacement);
                 Keyboard.Type(decl.Replacement);
-                Keyboard.Type("\t");
+                Keyboard.Type("→");

```

Comment:
You can use ExecuteCommand('Edit.CompleteWord') here instead. Also let's change the rest of the tests to be consistent if we're going to change it in this one spot.